numpy/core/defchararray.py

```diff
@@ -1785,6 +1785,26 @@ def isnumeric(a):
     --------
     unicode.isnumeric
 
+    Examples
+    --------
+    >>> np.char.isnumeric('¼')
+    array(True)
+
+    >>> np.char.isnumeric('123')
+    array(True)
+
+    >>> np.char.isnumeric('abc')
+    array(False)
+
+    >>> np.char.isnumeric('Ⅷ')
+    array(True)
+
+    >>> np.char.isnumeric(['Ⅷ', '㈥', '9.0', '1/4', '123abc'])
+    array([ True,  True, False, False, False])
```

Comment:
The output here is wrong - all of these inputs should result in `False`. It might be good to add at least one example that results in `True`:

```suggestion
    >>> np.char.isnumeric(['123', '123abc', '9.0', '1/4', 'VIII'])
    array([ True,  False, False, False, False])
```

Reply:
I can make the suggested changes, but when I run `np.char.isnumeric(['Ⅷ', '㈥', '9.0', '1/4', '123abc'])` the first two outputs are `True`
<img width="643" alt="image" src="https://user-images.githubusercontent.com/25092182/191595245-86f47418-efac-4ef7-aa02-99c9f0e2ffe7.png">